eth-types/src/geth_types.rs

```diff
@@ -90,26 +90,35 @@ impl BlockConstants {
 /// Definition of all of the constants related to an Ethereum transaction.
 #[derive(Debug, Default, Clone, Serialize)]
 pub struct Transaction {
-    /// From Address
+    /// Sender address
     pub from: Address,
-    /// To Address
+    /// Recipient address (None for contract creation)
     pub to: Option<Address>,
-    /// Nonce
+    /// Transaction nonce
     pub nonce: Word,
-    /// Gas Limit
+    /// Gas Limit / Supplied gas
     pub gas_limit: Word,
-    /// Value
+    /// Transfered value (None for no transfer)
```

Comment:
Not an `Option` so I suppose that you would say zero instead `None`

Reply:
Updated in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/484/commits/3bb4063976066d07d6d654dd82ffa77c1007cdb3